pyproject.toml

```diff
@@ -0,0 +1,51 @@
+[build-system]
+requires = [
+    "poetry>=0.12",
+]
+build-backend = "poetry.masonry.api"
+
+[tool.isort]
+profile = "black"
+
+[tool.poetry]
+name = "rle-array"
+description = "Run-length encoded pandas."
+authors= [
```

Comment:
Change to BlueYonder?

Reply:
I'm currently clarifying this, let's do that in another PR.

Reply:
Ah, got an answer for legal, fixed :)